config/web/translator-gridview.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+use Yiisoft\Aliases\Aliases;
+use Yiisoft\Translator\CategorySource;
+use Yiisoft\Translator\Message\Php\MessageSource;
+use Yiisoft\Translator\MessageFormatterInterface;
+
+return [
+    'translator.gridview' => static function (Aliases $aliases, MessageFormatterInterface $messageFormatter) {
+        $messageReader = new MessageSource($aliases->get('@gridview/resources/message'));
```

Comment:
Let's move translations to match what's used in app templates:

```suggestion
        $messageReader = new MessageSource($aliases->get('@gridview/resources/messages'));
```

Reply:
done.